app/models/translatable.rb

```diff
@@ -0,0 +1,14 @@
+module Translatable
+  def en_name
+    self.name
+  end
+
+  def localized_name
```

Comment:
Indent the `when` at the same level as `case` and instead of `||`ing, have `name` as the `else` within the `case`. E.g.

```ruby
case I18n.locale
when :en
  en_name
when :cy
  cy_name
else
  name
end
```

Also, it may be more defensive to explicitly bomb out if you hit the `else` clause. These kind of silent, soft-failures can cause surprises.